src/cmds/restic/cmd_backup.go

```diff
@@ -264,7 +272,7 @@ func readBackupFromStdin(opts BackupOptions, gopts GlobalOptions, args []string)
 		return err
 	}
 
```

Comment:
`os.Hostname` does quite a bit of stuff internally... Not like we're going to die if `os.Hostname` spends extra 60µs doing its job, but what if we only load hostname if empty here?